content/applications/marketing/events/overview/create.rst

```diff
@@ -0,0 +1,298 @@
+
+.. image:: create/events-invite-button.png
+   :align: center
+   :alt: View of an event form and the invite button in Odoo Events.
+
+This opens an invite template form, that can be emailed or sent out as an SMS. The message can be
+fully customized, and recipients can be added.
+
+Adding a :guilabel:`Subject` line for the invite message is required, but the
+:guilabel:`Preview Text` field is optional. The :guilabel:`Preview Text` is a catchy preview
+sentence meant to encourage recipients to open the email.
+
+.. note::
+   In most cases, the :guilabel:`Preview Text` is displayed next to the subject. Keep this field
+   empty to ensure the first characters of the email content appear, instead.
+
+Select invitees and configure recipient filters
+-----------------------------------------------
+
+Click the :guilabel:`Recipients` field, to reveal a drop-down menu of recipient options. These
+choices represent where Odoo will find the desired recipients' information.
+
+Once an option from this menu is selected (e.g. *Applicant*, *Contact*, *Event Registration*,
+*Lead/Opportunity*, etc.), Odoo can send it to all recipients who match that initial rule, or a
+rule(s) can be added (and configured) to narrow down the target recipients, by clicking
+:guilabel:`Add Filter`.
+
+.. image:: create/add-filter-button.png
+   :align: center
+   :alt: View of the add filter button beneath the recipients field in Odoo Events.
+
+Clicking :guilabel:`Add Filter` reveals three fields, formatted like an equation. To reveal the
+sub-menu options, click each field, and make the desired selections, until the preferred
+configuration has been acheived. The number of *Records* that match the rule(s) are indicated to the
+right of the :guilabel:`Recipients` field, in green.
+
+.. image:: create/recipient-filters.png
+   :align: center
+   :alt: View of the recipient filter fields used for event invite messages.
+
+.. note::
+   Some sub-menu options in the first rule field allow for a second choice to provide even more
+   specificity.
+
+To the right of each rule, is an :guilabel:`X`, a :guilabel:`+`, and :guilabel:`...`. The
+:guilabel:`X` deletes a specific node (line) of the rule. The :guilabel:`+` adds a node (line) to
+the rule. And, the :guilabel:`...` adds a branch to the node. This means two additional, indented
+sub-nodes are added to the rule. These sub-nodes are additional lines that provide even more
+specificity related to the line above it.
+
+Build a custom event invite
+---------------------------
+
+In the :guilabel:`Mail Body` tab, there are a number of pre-configured message templates to choose
+from. Select the desired template, and modify every element of its design details with Odoo's
+drag-and-drop building blocks (located on the right sidebar).
+
+.. image:: create/event-email-building-blocks.png
+   :align: center
+   :alt: View of the drag and drop building blocks used to customize event invite emails.
```

Comment:
Ah, interesting. I was told previously to avoid GIFs, because they're not "responsive" on other devices. But I think a GIF would be best, especially to showcase the speed (and simplicity) of the drag-and-drop action.

Here's a link to a screencast that could (potentially) be used to generate a GIF for this section: https://drive.google.com/file/d/16uIMJWoCNFo6tD1YDVzv3x5b0QxWcUlR/view?usp=sharing

Reply:
Here's a fresh screenshot, too (just in case the GIF idea doesn't pan out): https://drive.google.com/file/d/1cxs8nd1NWwihPMVb8BmLqI_A5pwV9-Av/view?usp=sharing

Reply:
Feel free to pick either and place it in the doc